template/rwa-onboarding/Goerli-DssSpell.t.sol

```diff
@@ -20,466 +18,834 @@ pragma solidity 0.6.12;
+
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done(), "TestError/spell-not-done");
+
+        checkSystemValues(afterSpell);
+
+        checkCollateralValues(afterSpell);
+    }
+
+    function testRemoveChainlogValues() private { // make public to use
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        // try chainLog.getAddress("XXX") {
+        //     assertTrue(false);
+        // } catch Error(string memory errmsg) {
+        //     assertTrue(cmpStr(errmsg, "dss-chain-log/invalid-key"));
+        // } catch {
+        //     assertTrue(false);
+        // }
+
+    }
+
+    function testCollateralIntegrations() private {
         vote(address(spell));
         scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
         // Insert new collateral tests here
-        checkIlkIntegration(
-            "RWA999-A",
-            GemJoinAbstract(addr.addr("MCD_JOIN_RWA999_A")),
-            ClipAbstract(addr.addr("MCD_CLIP_RWA999_A")),
-            addr.addr("PIP_RWA999"),
-            false,
-            false,
-            false
-        );
+        // checkIlkIntegration(
+        //     "TOKEN-X",
+        //     GemJoinAbstract(addr.addr("MCD_JOIN_TOKEN_X")),
+        //     ClipAbstract(addr.addr("MCD_CLIP_TOKEN_X")),
+        //     addr.addr("PIP_TOKEN"),
+        //     true,
+        //     true,
+        //     false
+        // );
+
+
+    }
+
+    function testLerpSurplusBuffer() private { // make public to use
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        // Insert new SB lerp tests here
+
+        LerpAbstract lerp = LerpAbstract(lerpFactory.lerps("NAME"));
+
+        uint256 duration = 210 days;
+        hevm.warp(block.timestamp + duration / 2);
+        assertEq(vow.hump(), 60 * MILLION * RAD);
+        lerp.tick();
+        assertEq(vow.hump(), 75 * MILLION * RAD);
+        hevm.warp(block.timestamp + duration / 2);
+        lerp.tick();
+        assertEq(vow.hump(), 90 * MILLION * RAD);
+        assertTrue(lerp.done());
     }
 
-    function testNewChainlogValues() public {
+    function testNewChainlogValues() public { // make private to disable
         vote(address(spell));
         scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
-        // RWA999
-        checkChainlogKey("RWA999");
-        checkChainlogKey("PIP_RWA999");
-        checkChainlogKey("MCD_JOIN_RWA999_A");
-        checkChainlogKey("RWA999_A_URN");
-        checkChainlogKey("RWA999_A_OUTPUT_CONDUIT");
-        checkChainlogKey("RWA999_A_INPUT_CONDUIT");
+        // RWAXXX
+        checkChainlogKey("RWAXXX_A_JAR");
+        checkChainlogKey("RWAXXX");
+        checkChainlogKey("MCD_JOIN_RWAXXX_A");
+        checkChainlogKey("RWAXXX_A_URN");
+        checkChainlogKey("RWAXXX_A_INPUT_CONDUIT_URN");
```

Comment:
⚠️⚠️⚠️ 

Here we might have an issue because deals that don't use USDC only have 1 input conduit for the urn and it's named `RWAXXX_A_INPUT_CONDUIT`. Maybe we should adopt `RWAXXX_A_INPUT_CONDUIT_URN` regardless moving forward, but then we'd have to retroactively update all other deals to this new naming conventions.